bigquery/README.md

```diff
@@ -4,3 +4,37 @@ This notifier pushes build data to a BigQuery instance.
 
 *Alpha Feature - Under Development*
 
+SQL queries through the BigQuery CLI:
+
+bq query '\<SQL QUERY\>'
+
+Legacy SQL dialect is set on default for the BigQuery CLI and must be disabled for the example queries to work.
+This can be done by editing ~/.bigqueryrc and adding the following lines:
+
+```
+[query]
+--use_legacy_sql=false
+```
+
+Example Queries:
+```
+# Listing overall build history
+
+SELECT * FROM `dataset.table`
+
+# Aggregating Build Counts by Status
```

Comment:
Only capitalize the first word of a heading (unless other words are official names or proper nouns)
https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings